### PR TITLE
Add section for graylog integration

### PR DESCRIPTION
**Pull request by fkuster**

### CODE REVIEW

File: Logging.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    ```Ruby
+      config.graylog_host = nil
+    ```
+
+* Add `semantic_logger.rb` in initializers folder.
+
+  * Example
+
+    ```Ruby
+      SemanticLogger.add_appender(
```
Comment (d4be4st) [inline Logging.md]: you are missing an `if Rails.application.config.graylog_host` here

Comment (fkuster) [inline Logging.md]: ty, i was to lazy to check it in feature branch :) my bad

File: Logging.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        SemanticLogger.add_appender(
+          appender:    :graylog,
+          url:         "udp://#{Rails.application.config.graylog_host}.infinum.co:12201",
+          application: "application_name-#{Rails.env}",
+          level:       :info
+        )
+      end
+    ```
+
+* Customize payload
```
Comment (d4be4st) [inline Logging.md]: Where to put this?

File: Logging.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+  * Add following code in the controllers which requests you want to track.
+
+    ```Ruby
+      def append_info_to_payload(payload)
+        super
+        payload[:ip_address] = request.remote_ip if request.remote_ip.present?
+        payload[:user] = current_user.username if current_user.present?
+      end
+    ```
+  * If there is more then one controller which requests you want to track, create `Loggable.rb` concern and include it in all controllers.
```
Comment (d4be4st) [inline Logging.md]: Leave this out.

File: Logging.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+          url:         "udp://#{Rails.application.config.graylog_host}.infinum.co:12201",
+          application: "application_name-#{Rails.env}",
+          level:       :info
+        )
+      end
+    ```
+
+* Customize payload
+
+  * Add following code in the controllers which requests you want to track.
```
Comment (d4be4st) [inline Logging.md]: `which logs you wish to alter`

**Pull request merged**